examples/next-monaco/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
```

Comment:
Could you remove the duplicates here? There should be one folder, `monaco-react`.

Reply:
is it this way @leerob 

```ts
{
  "exclude": ["node_modules", "monaco-react"]
}
```

Reply:
> I mean that there's two examples, two folders, in this PR. We should just have one 😄

sorry i missed this, i'll fix it now